go/vt/srvtopo/discover_test.go

```diff
@@ -84,28 +81,34 @@ func TestFindAllTargets(t *testing.T) {
 				},
 			},
 		},
-	}); err != nil {
-		t.Fatalf("can't add srvKeyspace: %v", err)
-	}
+	}), "can't add srvKeyspace: %v", err)
```

Comment:
This is printing the output of `err.Error()` — rather than the ignored return value from the `ts.UpdateSrvKeyspace()` call — but that variable was set to the return value from `FindAllTargets()` above. I think that just using `assert.NoErorr()` instead is fine.

Reply:
Makes sense, `testify`'s output makes things pretty clear already 👍